<!DOCTYPE HTML>
<!--
	Massively by HTML5 UP
	html5up.net | @ajlkn
	Free for personal and commercial use under the CCA 3.0 license (html5up.net/license)
-->
<html>
	<head>
		<title>Studying mitochondrial DNA</title>
		<meta charset="utf-8" />
		<meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
		<link rel="stylesheet" href="assets/css/main.css" />
		<link rel="shortcut icon" href="favicon.ico" type="image/x-icon">
		<link rel="icon" href="favicon.ico" type="image/x-icon">
		<noscript><link rel="stylesheet" href="assets/css/noscript.css" /></noscript>
		<script type="text/javascript" async
		src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.4/latest.js?config=TeX-MML-AM_CHTML">
		</script>
		<script type="text/x-mathjax-config">
		MathJax.Hub.Config({
			tex2jax: {inlineMath: [['$','$'], ['\\(','\\)']]}
			});
		</script>
		<script type="text/javascript" async src="path-to-mathjax/MathJax.js?config=TeX-AMS_CHTML"></script>
	</head>
	<body class="is-loading">

		<!-- Wrapper -->
			<div id="wrapper" class="fade-in">

				<!-- Intro -->
					<div id="intro">
						<h1>Investigating mitochondrial DNA quantification and parametric model selection</h1>
						<p>by Heather Jackson</p>
						<ul class="actions">
							<li><a href=abstracts.html class="button big">Read Abstracts</a></li>
						</ul>
					</div>

				<!-- Header -->
					<header id="header">
						<a href="abstracts.html" class="logo">Read abstracts</a>
					</header>

				<!-- Nav -->
					<nav id="nav">
						<ul class="links">
							<li class="active"><a href="index.html">Introduction</a></li>
							<li><a href="scrna.html">Single-cell RNA-seq data</a></li>
							<li><a href="modelling.html">Modelling copy number</a></li>
							<li><a href="summary.html">Summary</a></li>
							<li><a href="references.html">References</a></li>
							<li><a href="acknowledgements.html">Glossary</a></li>
						</ul>
					</nav>

				<!-- Main -->
					<div id="main" class="alt">

						<!-- Featured Post -->
						<section class="post">
							<div class="inner">
								<header class="major">
									<h1>Introduction</h1></header>
									<h2>Mitochondria</h2>
									<p><a href=acknowledgements.html>Mitochondria</a> are <a href=acknowledgements.html>organelles</a> found in eukaryotic cells. They are responsible for energy transduction and metabolism through a process called <a href=acknowledgements.html>oxidative phosphorylation</a> <a href=references.html>(Stewart & Chinnery, 2015; Taanman, 1999)</a>. Mitochondria are thought to have arisen due to a proteobacterium becoming associated with and endocytosed by a prokaryotic cell, leaving a double-membraned organelle <a href=references.html>(Stewart & Chinnery, 2015)</a>.  Due to multiple passages over time of genetic material from the endocytosed organelle into the prokaryotic cell nucleus, the genome of the organelle shrank in size. The end result was mitochondria and their autonomously replicating DNA genome of 16.5 kilo-base pairs, as shown in figure 1 <a href=references.html>(Anderson et al., 1981; Stewart & Chinnery, 2015)</a>.
	Over time, various differences have arisen between nuclear DNA (nDNA) and <a href=acknowledgements.html>mitochondrial DNA</a> (<a href=acknowledgements.html>mtDNA</a>). For example, mtDNA is packaged in a circular coding arrangement with multiple independently-proliferating copies present in each cell <a href=references.html>(Stewart & Chinnery, 2015)</a>. Mitochondrial proteins are encoded by nuclear and mitochondrial genes, meaning that the mitochondrion is dependent on both genetic systems in order to function healthily <a href=references.html>(Anderson et al., 1981; Stewart & Chinnery, 2015)</a>.</p>

							<figure>
								<center>
								<a class="image-link" href="images/mtdnagenome.png" alt="<strong>Figure 1.">
									<img src="images/mtdnagenome.png" style="width: 40%;"></a>
									<figcaption class="img-comment" style="margin: auto; text-align: center;"><font size="-1"><strong>Figure 1: </strong>The circular arrangement of the human mitochondral genome showing the two ribosomal RNAs (12S and 16S), the 22 transfer RNAs (single letter abbreviations) and all coding genes. This figure is taken from <a href=references.html>Yoon, Koob and Yoo (2010)</a>.</font></figcaption>
								</center>
							</figure>
							<p>
							<h2>Motivation for studying mitochondria</h2>
							<p>Mitochondrial diseases are a wide range of disorders that arise due to mitochondria functioning abnormally <a href=references.html>(Noyazov, Kahler & Frye, 2016)</a>. Primary mitochondrial diseases, such as Leigh syndrome or MELAS, are caused by mutations in mtDNA or nDNA that affect oxidative phosphorylation and prevent mitochondria from fulfilling their function <a href=references.html>(Noyazov, Kahler & Frye, 2016)</a>. These diseases are caused by germline mutations, however somatic, deleterious mutations in mtDNA can also emerge. Somatic mutations have been reported to arise at high frequencies and have been associated with conditions such as cancer and neuro-degenerative diseases <a href=references.html>(Gemmell, Metcalf & Allendorf, 2004; Ruiz-Pesini et al., 2000; Wallace, 1999)</a>. An understanding of how mutations can arise and propogate themselves through a population is pivotal for attempting to treat or cure mitochondrial disorders. Furthermore, mitochondrial dysfunction has been identified as a hallmark of ageing <a href=references.html>(Lopez-Otin et al. 2013)</a>. By studying mitochondria, a better insight into the ageing process could be acquired. This insight	 could be used to identify pharmaceutical targets to ameliorate human health during ageing <a href=references.html>(Lopez-Otin et al. 2013)</a>.</p>
						<p>
							<h2>Heteroplasmy</h2>
							<a href=acknowledgements.html>Heteroplasmy</a> describes the mixture of <a href=acknowledgements.html>wild type</a> and <a href=acknowledgements.html>mutant mtDNA</a> within a single cell. It is calculated as follows: $H = m/(w+m)$ where $m$ represents mutant mtDNA and $w$ represents wild type mtDNA. Heteroplasmic mutations can affect multiple organ systems <a href=references.html>(Stewart & Chinnery, 2015)</a>. Heteroplasmy levels vary greatly between cells and correlate with the severity of the phenotype and the level of organ vulnerability and involvement <a href=references.html>(DiMauro et al., 2013; Stewart & Chinnery, 2015)</a>. Variants with low mutational burdens can accumulate to high levels over time resulting in biochemical defects once a threshold has been surpassed, as shown in figure 2 <a href=references.html>(Brierley et al., 2004; Stewart & Chinnery, 2015)</a>. Studying heteroplasmy is essential for understanding how mitochondrial dysfunction can arise and how it contributes to factors such as ageing.
						</p>
						<p>
							<figure>
								<center>
								<a class="image-link" href="images/cropped_het.png" alt="<strong>Figure 2.">
									<img src="images/cropped_het.png" style="width: 70%;"></a>
									<figcaption class="img-comment" style="margin: auto; text-align: center;"><font size="-1"><strong>Figure 2: </strong>A schematic to demonstrate how different levels of heteroplasmy can affect the phenotype of the cell. Once heteroplasmy levels exceed a certain threshold (shown by the dotted line) the phenotype, e.g. the biochemical defect, can be observed (i.e grey cell). This image has been adapted from <a href=references.html>Stewart and Chinnery (2015)</a>.</font></figcaption>
								</center>
							</figure>
						</p>
						Various experimental methods can be used to quantify heteroplasmy including DNA sequencing <a href=references.html>(Irwin et al., 2009)</a>, high-performance liquid chromatography <a href=references.html>(Meierhofer et al., 2005)</a> and pyrosequencing <a href=references.html>(Sazonova et al., 2009)</a>.</p>
						<p>
							<h2>Copy number</h2>
							<p>In order to maintain cellular function and energy requirements, the number of copies of mtDNA present in cells is tightly regulated so that the appropriate amount of ATP can be generated  <a href=references.html>(Dickinson et al., 2013; Stewart & Chinnery, 2015)</a>. mtDNA <a href=acknowledgements.html>copy number</a> varies greatly between different cell types. Cells with high-energy requirements, such as neurons or muscle cells, maintain higher copy numbers in comparison to those with low-energy requirements, such as endothelial or spleen cells  <a href=references.html>(Dickinson et al., 2013)</a>. Changes in mtDNA copy number have been observed in various cancers, for example mtDNA copy number depletion has been identified in bladder, kidney and breast cancers and has been associated with reduced expression of mitochondrial genes and activity  <a href=references.html>(Reznik et al., 2016)</a>. Studying mtDNA copy number in healthy cells will enable deviations from the norm to be more readily identified as copy number can serve as a surrogate marker for the cellâ€™s oxidative phosphorylation capacity <a href=references.html>(Reznik et al., 2016)</a>.  Furthermore, studying copy number in isolation of heteroplasmy is essential for gaining an insight into the mechanisms used by cells to control their mitochondrial populations <a href=references.html>(Johnston & Jones, 2016).	</p>
						<p>
							<h2>Project aims</h2>
							<center><li><a href=acknowledgements.html>Single-cell RNA-seq</a> characterises the transcriptomes of single cells. This information could be used to infer mitochondrial heteroplasmy and copy number. From this, the impacts that mitochondrial physiology have on the transcriptome, and vice versa, could be deduced. Single-cell RNA-seq data requires various processing steps. We will explore how raw expression data is normalised to account for systematic variability introduced by different experimental batches.</li></p>
							<li>The mechanisms used by cells to control their mitochondria is important for understanding how mtDNA mutations are able to proliferate and accumulate over time. Knowledge of the rate at which mtDNA is replicated can help to inform us about such control mechanisms. We will apply an <i>in silico</i> approach and will simulate two alternative models, each with different replication rate functions, and explore whether it is feasible to distinguish between the two models given a single-cell dataset of mtDNA copy number. </li></center></p>

							</div>
								<ul class="actions">
									<li><a href="#" class="button big">Back to top</a></li>
								</ul>
							</article>
							</section>


						<!-- Footer -->

					</div>
				<!-- Copyright -->
					<div id="copyright">
						<ul><li>Heather Jackson</li><li>CID: 01402152</li><li>Imperial College London</li>
							<li>&copy; Design: <a href="https://html5up.net">HTML5 UP</a></li></ul>
							<ul><li>Background: <a href="https://phys.org/news/2017-06-mitochondria-blood-cell-formation.html">Northwestern University</a></li></ul>

					</div>

			</div>

		<!-- Scripts -->
			<script src="assets/js/jquery.min.js"></script>
			<script src="assets/js/jquery.scrollex.min.js"></script>
			<script src="assets/js/jquery.scrolly.min.js"></script>
			<script src="assets/js/skel.min.js"></script>
			<script src="assets/js/util.js"></script>
			<script src="assets/js/main.js"></script>

	</body>
</html>
